Sources/TensorFlow/Operators/Math.swift

```diff
@@ -644,6 +644,15 @@ public func rsqrt<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
     Raw.rsqrt(x)
 }
 
+/// Returns the cosine similarity between x and y.
```

Comment:
Please add backquotes around all parameters (or generally any program identifiers) in API documentation comments.
```suggestion
/// Returns the cosine similarity between `x` and `y`.
```